docs/development.rst

```diff
@@ -26,12 +26,12 @@ details and :pull:`154` for an example of how the process goes). This isn't
 Release Policy
 ==============
 
-Hypothesis releases follow `semantic versioning <http://semver.org/>`_.
+Hypothesis releases follow `semantic versioning <https://semver.org/>`_.
 
 We maintain backwards-compatibility wherever possible, and use deprecation
 warnings to mark features that have been superseded by a newer alternative.
-If you want to detect this, the :py:meth:`strict setting <hypothesis.settings.strict>`
-upgrades all Hypothesis warnings to errors.
+If you want to detect this, you can upgrade warnings to errors in the usual
+ways.
```

Comment:
I think this should have a link to https://docs.python.org/3.1/library/warnings.html#the-warnings-filter